www/src/components/docs-table-of-contents.js

```diff
@@ -17,38 +17,40 @@ function isUnderDepthLimit(depth, maxDepth) {
 function createItems(items, location, depth, maxDepth) {
   return (
     items &&
-    items.map(item => (
+    items.map((item, index) => (
       <li
         sx={{ [mediaQueries.xl]: { fontSize: 1 } }}
-        key={location.pathname + item.url}
+        key={location.pathname + (item.url || depth + `-` + index)}
```

Comment:
```
 key="/docs/production-app/2-0"
   key="/docs/production-app/3-0"
```

i have done this to be sure to have an uniue key, but maybe there are never more than one `item` there for empty headers so i can leave the undefined here - but with `undefined` it looks like an error.

```
  key="/docs/production-app/undefined"
    key="/docs/production-app/undefined"
```

Reply:
I think this is good idea - let's keep it!